uint16 gpioId
uint16 frequency
---
bool result